lib/src/hashgraph/mirror-node-client.ts

```diff
@@ -0,0 +1,22 @@
+import { NetworkType } from "../types";
+
+export class MirrorNodeClient {
+  url = "https://mainnet-public.mirrornode.hedera.com/";
```

Comment:
My main issue with this is here - the public mirror node is not meant to be used in production

Reply:
Ah I didn't realize that. I see now that the production-ready solutions are DragonGlass (paid) and Kabuto (in open beta, but maybe will require payment in the future).

Then the best thing to do here would be to send the account's public key over from hashpack to the browser via hashconnect via the websocket. Either as an ApprovePairing message or a new message entirely.

Reply:
Guess that relates to #90